src/components/map/MapPage.js

```diff
@@ -62,13 +64,13 @@ const MapPage = () => {
           swlng: bounds.sw.lng,
           muni: filters.muni ? 1 : 0,
           t: filters.types.toString(),
+          limit: 250,
```

Comment:
should we move this constant `{...limit: 250}` to a utils file or is keeping it local good for now?

Reply:
Will eventually consolidate all getLocations requests in a file in utils!